query/control/controller_test.go

```diff
@@ -145,21 +145,6 @@ func TestController_QuerySuccess(t *testing.T) {
 			if err := q.Err(); err != nil {
 				t.Errorf("unexpected error: %s", err)
 			}
-
-			stats := q.Statistics()
```

Comment:
We already check for stats handling in `TestController_QueryRuntimeError`, we don't need another check here